cmd/hook.go

```diff
@@ -376,7 +376,12 @@ Gitea or set your environment appropriately.`, "")
 		oldCommitIDs[count] = string(fields[0])
 		newCommitIDs[count] = string(fields[1])
 		refFullNames[count] = git.RefName(fields[2])
-		if refFullNames[count] == git.BranchPrefix+"master" && newCommitIDs[count] != git.EmptySHA && count == total {
+
+		hash, err := git.HashFromString(newCommitIDs[count])
+		if err != nil {
+			hash = &git.Sha1Hash{}
```

Comment:
It's better to just `return err`

Reply:
Re-reading this, your comment makes perfect sense and I have no idea who wrote that patch anymore :rofl:

Changing this and also the changes below in the hook.go